Objective-C/Tests/ReplicatorTest+Collection.m

```diff
@@ -19,26 +19,146 @@
 
 #import "ReplicatorTest.h"
 
+#ifdef COUCHBASE_ENTERPRISE
 @interface ReplicatorTest_Collection : ReplicatorTest
 
 @end
 
-@implementation ReplicatorTest_Collection
+@implementation ReplicatorTest_Collection {
+    CBLDatabaseEndpoint* _target;
+    CBLReplicatorConfiguration* _config;
+}
 
 - (void)setUp {
     [super setUp];
+    
+    _target = [[CBLDatabaseEndpoint alloc] initWithDatabase: self.otherDB];
+    _config = [[CBLReplicatorConfiguration alloc] initWithTarget: _target];
 }
 
 - (void)tearDown {
+    _target = nil;
+    _config = nil;
+    
```

Comment:
Are these necessary?

Reply:
Yes, since the `_config` is a property, the collections which are saved inside it will not be released as long as the property exists or we might need to release by setting it to nil